integration/presto/src/main/java/org/apache/carbondata/presto/CarbonVectorBatch.java

```diff
@@ -0,0 +1,112 @@
+package org.apache.carbondata.presto;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+public class CarbonVectorBatch {
+
+  private static final int DEFAULT_BATCH_SIZE = 1024;
```

Comment:
why is 1024?  for byteArr = new byte[batchSize];  whether can set smaller,or not ?

Reply:
The batchSize is the size of the complete batch so for every Column we have to set it as byteSize , it is the number of records in the batch so it has to be equivalent to the batch size